### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by raulriera**
## What

Just some test, changed the text tho.

### CODE REVIEW

**Comment by krbarnes** [comment]
No

**Comment by raulriera** [comment]
test

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-TEST
+## Probot test
+
+I will be using this repo to test everything pro bot related. Cool story, bro
```
Comment (dannbeaus) [inline README.md]: I took 3 commits for this?

**Comment by krbarnes** [comment]
~Minimun~ Minimum

**Comment by dannbeaus** [comment]
🤣 ~approval~ @krbarnes

**Comment by dannbeaus** [comment]
author's choice https://github.com/apps/minimun-reviews

**Pull request closed without merging**